docs/reference/cat/segments.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[cat-segments]]
+== cat segments
+
+The `segments` command provides low level information about the segments
+in the shards of an index. It provides information similar to the
+link:indices-segments.html[_segments] endpoint.
+
+[source,shell]
+--------------------------------------------------
+% curl 'http://localhost:9200/_cat/segments?v'
+index shard prirep ip            segment generation docs.count [...]
+test  4     p      192.168.2.105 _0               0          1
+test1 2     p      192.168.2.105 _0               0          1
+test1 3     p      192.168.2.105 _2               2          1
+--------------------------------------------------
+
+[source,shell]
+--------------------------------------------------
+[...] docs.deleted  size size.memory committed searchable version compound
+                 0 2.9kb        7818 false     true       4.10.2  true
+                 0 2.9kb        7818 false     true       4.10.2  true
+                 0 2.9kb        7818 false     true       4.10.2  true
+--------------------------------------------------
+
+The output shows information about index names and shard numbers in the first
+two columns.
+
+If you only want to get information about segments in one particular index,
+you can add the index name in the URL, for example `/_cat/segments/test`. Also,
+several indexes can be queried like `/_cat/segments/test,test1`
+
+
+The following columns provide additional monitoring information:
+
+prirep::        Whether this segment belongs to a primary or replica shard.
+
+ip::            The ip address of the segments shard.
+
+segment::       A segment name, derived from the segment generation. The name
+                is internally used to generate the file names in the directory
+                of the shard this segment belongs to.
+
+generation::    The generation number is incremented with each segment that is written.
+                The name of the segment is derived from this generation number.
+
+docs.count::    The number of non-deleted documents that are stored in this segment.
+
+docs.deleted::  The number of deleted documents that are stored in this segment.
+                It is perfectly fine if this number is greater than 0, space is
```

Comment:
Sure, can do. The current wording comes from .../docs/reference/indices/segments.asciidoc.